packages/upstream/angstrom.0.16.1/opam

```diff
@@ -1,20 +1,15 @@
 opam-version: "2.0"
-name: "angstrom"
-version: "0.16.0"
-synopsis: "Parser combinators built for speed and memory-efficiency"
-description: """\
-Angstrom is a parser-combinator library that makes it easy to write efficient,
-expressive, and reusable parsers suitable for high-performance applications. It
-exposes monadic and applicative interfaces for composition, and supports
-incremental input through buffered and unbuffered interfaces. Both interfaces
-give the user total control over the blocking behavior of their application,
-with the unbuffered interface enabling zero-copy IO. Parsers are backtracking by
-default and support unbounded lookahead."""
 maintainer: "Spiros Eliopoulos <spiros@inhabitedtype.com>"
-authors: "Spiros Eliopoulos <spiros@inhabitedtype.com>"
-license: "BSD-3-Clause"
+authors: [ "Spiros Eliopoulos <spiros@inhabitedtype.com>" ]
+license: "BSD-3-clause"
```

Comment:
The SPDX identifier shouldn't be changed, it will make the CI to fail. The correct identifier was being used previously: https://spdx.org/licenses/BSD-3-Clause.html

Reply:
Although https://spdx.org/licenses/BSD-3-clause.html redirects to https://spdx.org/licenses/BSD-3-Clause.html, so I think both might be valid.

Reply:
This says https://github.com/david-a-wheeler/spdx-tutorial they are case insensitive.

Reply:
Opened PR here to see if we can match these in case insensitive way https://github.com/xapi-project/xs-opam/pull/697

Reply:
I'll have to change the CI to match the case insensitiveness.

It's tricky to add this changer without adding churn to the licenses, both here and the .spec file (because git is case-sensitive)